website/more_documentation/button_documentation.py

```diff
@@ -1,4 +1,9 @@
+from asyncio import sleep
+from contextlib import asynccontextmanager
+
 from nicegui import ui
+from nicegui.elements.button import Button
```

Comment:
This import is not needed, since you can use `ui.button` for the type annotation.

Reply:
Done